vanilla-framework/vanilla-framework
# Delete typo scale demo page

## Done

Each of these patterns has their own demo so this collection is not needed.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/
- Verify no errors
